src/main/java/address/controller/TagListController.java

```diff
@@ -21,9 +21,9 @@
     ModelManager modelManager;
 
     @FXML
-    private AnchorPane mainPane;
+    private AnchorPane tagListMainPane;
     @FXML
-    private ScrollPane tags;
+    private ScrollPane tagListTags;
```

Comment:
It is hard to distinguish between these two objects based on their names. Both are Panes but you used `Pane` in only one name.